src/Microsoft.DotNet.Build.Tasks/PackageFiles/crossgen.sh

```diff
@@ -78,7 +82,7 @@ __targetDir=$1
 __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
 __toolsDir=$__scriptpath/../Tools
 __dotnet=$__toolsDir/dotnetcli/dotnet
-__packagesDir=$__scriptpath/../packages
+__packagesDir="${NUGET_PACKAGES:-${__scriptpath}/../packages}"
```

Comment:
In that case does it make since to stop hardcoding the coreclr package version and instead use the shared framework version to restore Microsoft.NETCore.App instead?